### ISSUE TITLE
Crash occurs in NavigationMapView if it was created in storyboard.

### ISSUE THREAD

**Issue by MaximAlien** [mapbox/mapbox-navigation-ios#2813]
**Mapbox Navigation SDK version:** `v2.0.0-alpha.1`

### Steps to reproduce

 1. Open Example application provided by Navigation SDK for iOS.
 2. Request route.
 3. Start Custom UI navigation.

### Actual behavior

Crash occurs right before starting navigation when accessing `MapView` properties.

### Expected behavior

Crash should not occur and navigation session should start.

Issue is currently blocked as it occurs on Maps SDK side.

**Comment by 1ec5** [comment]
```
Thread 1 Queue : com.apple.main-thread (serial)
#0    0x0000000102ac32c1 in ___lldb_unnamed_symbol5951$$MapboxMaps ()
#1    0x0000000102ac2d2c in ___lldb_unnamed_symbol5950$$MapboxMaps ()
#2    0x0000000102d7c204 in NavigationMapView.showsUserLocation.setter at /path/to/MapboxNavigation/Sources/MapboxNavigation/NavigationMapView.swift:173
#3    0x0000000102d807be in NavigationMapView.commonInit() at /path/to/MapboxNavigation/Sources/MapboxNavigation/NavigationMapView.swift:259
#4    0x0000000102d8067b in NavigationMapView.init(coder:) at /path/to/MapboxNavigation/Sources/MapboxNavigation/NavigationMapView.swift:251
#5    0x0000000102d80703 in @objc NavigationMapView.init(coder:) ()
```

**Comment by 1ec5** [comment]
Fixed in mapbox/mapbox-navigation-ios#2835.
